lesson6/car.rb

```diff
@@ -0,0 +1,9 @@
+cars = ['Toyota', 'Fiat', 'Renault', 'Pegeout', 'Kia', 'Daewoo', 'Honda', 'Mazda']
+puts 'У нас есть 8 машиин. Вам какую?'
+a = gets.chomp.to_i - 1
+
+if a >= 0 && a <= 7 #a == 1 || 2 || 3 || 4 || 5 || 6 || 7 || 8
```

Comment:
a == 1 || a == 2 || a == 3 || a == 4...

but pls, remove comment, it's not used anymore